test/runtests.jl

```diff
@@ -91,7 +93,9 @@ try
     NEW = GZip.GZLIB_VERSION > "1.2.3.9"
     pos = position(gzfile)
     NEW && (pos2 = position(gzfile,true))
-    @test_throws ErrorException seek(gzfile, 100)   # can't seek backwards on write
+    try
```

Comment:
I think only the error message is broken. 
seek(gzfile, 100) returns true in all cases, whether the current position can be changed or not.
To test the behaviour on Windows, we could check that the position remains unchanged after running seek(gzfile,100) or seek(gzfile, 1000000)
A more thorough approach might  be to actually throw an error in Windows too, around Gzip.jl:328.